package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Web application (server + web client) for VelociRaffle",
   "main": "index.js",
   "scripts": {
-    "build": "babel server -d serverDist --presets es2015",
+    "build": "babel server -d serverDist",
     "precommit": "jscs . --fix && eslint ./client/app && npm test",
     "postinstall": "npm run build",
     "dev-client": "gulp",
-    "dev-server": "nodemon server/index.js --exec babel-node --presets es2015",
-    "test": "karma start"
+    "dev-server": "env $(cat .env) nodemon server/index.js --exec babel-node",
```

Comment:
`env $(cat .env)` will load any environment variables that are in a `.env` file. I probably should change this command to use Foreman instead to be more cross-compatible between OS's.